examples/register.rs

```diff
@@ -1,18 +1,18 @@
-extern crate env_logger;
-extern crate libmdns;
+use env_logger;
+use libmdns;
 
-pub fn main() {
+#[tokio::main]
+pub async fn main() {
     env_logger::init();
 
-    let responder = libmdns::Responder::new().unwrap();
+    let responder = libmdns::Responder::builder().use_v6(false).build().unwrap();
+
     let _svc = responder.register(
         "_http._tcp".to_owned(),
         "libmdns Web Server".to_owned(),
         80,
         &["path=/"],
     );
```

Comment:
If we want to add more options, what do you think about a Builder for each service? Then extending the options for each published Service instead.